### PR TITLE
Correct a small mistake

### PR DESCRIPTION
**Pull request by julianfairfax**
## What does this PR do?

This corrects a small mistake in the docs

## Why is this change important?

It's not, but there's no reason not to make the change.

## How to test this PR locally?

I don't think you have to?

## Author's checklist

No notes

## Related issues

No issues for this are open as far as I know.

### CODE REVIEW

File: docs/src/index.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Source-Code
 ===========
 
-This is a partial documentation of our source code.  We are not aim to document
+This is a partial documentation of our source code. We are not aiming to document
```
Comment (return42) [inline docs/src/index.rst]: The double space after the dot of sentence end was intended. Typographic tools use the two spaces after the end of a sentence to distinguish from the period of an abbreviation.

**Review by return42 — CHANGES_REQUESTED** [review]
Thanks for your patch / I have a nit to remark

File: docs/src/index.rst (hunk 1 above)
Comment (julianfairfax) [reply docs/src/index.rst]: Oh oops, that's my mistake. Fixed.

**Review by return42 — APPROVED** [review]
Thanks!

**Pull request merged**